index.js

```diff
@@ -67,19 +100,10 @@ module.exports = (input, opts) => {
 	const runTests = !opts.yolo;
 	const runCleanup = !opts.skipCleanup && !opts.yolo;
 
-	if (VERSIONS.indexOf(input) === -1 && !semver.valid(input)) {
-		return Promise.reject(new Error(`Version should be either ${VERSIONS.join(', ')}, or a valid semver version.`));
-	}
-
 	const tasks = new Listr([
 		{
-			title: 'Prerequisite check',
-			task: () => execa.stdout('npm', ['version', '--json']).then(json => {
-				const versions = JSON.parse(json);
-				if (semver.gte(process.version, '6.0.0') && !semver.satisfies(versions.npm, '>=2.15.8 <3.0.0 || >=3.10.1')) {
-					return Promise.reject(new Error(`npm@${versions.npm} has known issues publishing when running Node.js 6. Please upgrade npm or downgrade Node and publish again. https://github.com/npm/npm/issues/5082`));
-				}
-			})
+			title: 'Checking versions',
```

Comment:
I think it should still be called `Prerequisite check`

`Checking versions` is confusing as it's checking both the version you supplied and the npm version, which are totally different things.